components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java

```diff
@@ -188,7 +188,7 @@ public static void validateDomain(String domainName) throws Exception {
         if (domainName.matches(ILLEGAL_CHARACTERS_FOR_TENANT_DOMAIN)) {
             String msg = "The tenant domain ' " + domainName +
                          " ' contains one or more illegal characters. the valid characters are " +
```

Comment:
In the log comment, "the valid" should be "The valid" and end the sentence with a dot.